.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
As this is updating, maybe go directly to 20 which are promoted to LTS in 3 weeks?

Reply:
that is for in 3 weeks 😄